scripts/daily.php

```diff
@@ -141,14 +175,13 @@ private function insertAll($table, $fields, $datas, $print = TRUE)
                 echo "Error inserting : " . $table . "\n" . $e->getMessage() . "\n";
                 die;
             }
-        } else {
-            echo "Nothing to insert in " . $table . "\n";
         }
     }
 
     public function fillDeputes()
     {
-        echo "fillDeputes starting \n";
+        $startTime = microtime(true); // Record the start time
```

Comment:
plus utile du coup?

Reply:
non mais cette variable n'est pas utilisée : $startTime = microtime(true); // Record the start time

Reply:
Ah yes, je vais nettoyer ça